.travis.yml

```diff
@@ -1,8 +1,41 @@
+---
 language: node_js
 node_js:
-  - "0.10.33"
-services: mongodb
+  - '0.10'
+
+sudo: required
+
+branches:
+  only:
+    - master
+    - develop
+
+git:
+  depth: 10
+
+services:
+  - mongodb
+
+cache:
+  directories:
+    - node_modules
+
 before_install:
-  - "npm install -g npm"
-  - "npm cache clean"
-  - "npm install -g adapt-cli"
+  - npm install -g npm
+  - npm install -g adapt-cli
+
+install:
+  - npm config set spin false
+  - npm install
+
+matrix:
+  fast_finish: true
+
+notifications:
+  email: false
```

Comment:
Thank you for explaining me this :+1:.
Previously I was thinking that people might not be interested in receiving the build status emails. Now I'm thinking about it may help them to check the status via email once build finished on Travis because sometime it takes time to get completed.

I have removed the email configuration, now it should work as you have mentioned above :)